docs/reference/ml/df-analytics/apis/evaluateresources.asciidoc

```diff
@@ -95,4 +96,31 @@ which outputs a prediction of values.
   
 `metrics`::
   (object) Specifies the metrics that are used for the evaluation. Available 
-  metrics are `r_squared` and `mean_squared_error`.
\ No newline at end of file
+  metrics are `r_squared` and `mean_squared_error`.
+  
+  
+[[classification-evaluation-resources]]
+==== {classification-cap} evaluation objects
+
+{classification-cap} evaluation evaluates the results of a {classanalysis} which 
+outputs a prediction that identifies to which of the classes each document 
+belongs.
+
+
+[discrete]
+[[classification-evaluation-resources-properties]]
+===== {api-definitions-title}
+
+`actual_field`::
+  (string) The field of the `index` which contains the ground truth. The data 
+  type of this field must be string.
+  
+`metrics`::
+  (object) Specifies the metrics that are used for the evaluation. Available 
+  metric is `multiclass_confusion_matrix`.
+  
+`predicted_field`::
+  (string) The field in the `index` that contains the predicted value, in other 
+  words the results of the {classanalysis}. The data type of this field is 
+  string. You need to add `.keyword` to the predicted field name. For example, 
```

Comment:
You may want clarify that the "predicted field name" is the one that the user put in classification analysis object as "prediction_field_name" (or the default value was assigned).